core/src/main/java/amino/run/app/DMSpec.java

```diff
@@ -80,12 +81,12 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         DMSpec dmSpec = (DMSpec) o;
-        return Objects.equals(name, dmSpec.name) && Objects.equals(configs, dmSpec.configs);
+        return Objects.equal(name, dmSpec.name) && Objects.equal(configs, dmSpec.configs);
```

Comment:
Why not simply generate 1.6 compliant equals and hashCode methods instead of this `com.google.common.base.Objects` 3rd party dependency.

Reply:
ok agreed